src/main/java/net/fabricmc/loader/metadata/BuiltinModMetadata.java

```diff
@@ -76,6 +76,11 @@ public String getId() {
 		return id;
 	}
 
+	@Override
+	public Collection<String> getAliases() {
+		return Collections.emptyList();
```

Comment:
Not in scope of this PR, but it may a neat thing to look into for a GameProvider's builtin mods to possibly provide aliases.